sw/airborne/modules/computer_vision/video_thread.c

```diff
@@ -80,6 +80,9 @@ static void *video_thread_function(void *data)
 {
   struct video_config_t *vid = (struct video_config_t *)data;
 
+  char print_tag[80];
+  sprintf(print_tag, "video_thread-%s", vid->dev_name);
```

Comment:
@guidoAI the image memory is freed after the computer vision pipeline function is finished, i.e. when all cv_functions are executed

sw/airborne/modules/computer_vision/video_thread.c:149